app.rb

```diff
@@ -0,0 +1,26 @@
+require 'sinatra'
+require './lib/takeaway'
+require './lib/text'
+
+$takeaway = Takeaway.new
```

Comment:
How would you now think about doing this instead of using the evil evil global variable?